src/appfl/algorithm/server_federated.py

```diff
@@ -0,0 +1,90 @@
+import logging 
```

Comment:
Should we have `fedavg` here?

Reply:
The ``fedavg`` is the one we had before we implement the adaptive federated optimization techniques (including fedavg, fedavgm, fedadagrad, fedadam, fedyogi).
Since we have ``server_fed_avg.py``, how about we remove ``fedavg.py``?

Reply:
agreed

Reply:
done